### DISCUSSION TITLE
Weekend Schedule in the app

### DISCUSSION THREAD

**Discussion by stefanverkerknl** [f1multiviewer/issue-tracker#79] [Feature requests] [answered]
There's one reason left why I visit F1TV.com: checking the weekend schedule with the timetable. For example: https://f1tv.formula1.com/page/4351/formula-1-lenovo-british-grand-prix-2022

Is it possible to add / embed this to the app? This idea obviously has a low priority. 

Thanks!

**Comment by f1multiviewer** [comment, marked as answer]
It's available in v1.4.1!
